public_html/wp-content/themes/pattern-directory/src/components/pattern-preview-actions/success-message.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, Notice } from '@wordpress/components';
+
+const SuccessMessage = ( { showMessage, onClick } ) => (
```

Comment:
Can this be renamed to `CopySuccessMessage` or similar? I thought it was generic at first, so a more specific name would be clearer.

Reply:
Yep!